src/generics/constants/api-responses.js

```diff
@@ -45,4 +45,6 @@ module.exports = {
 	ENTITY_ID_OR_LOCATION_ID_NOT_FOUND: 'ENTITY_ID_OR_LOCATION_ID_NOT_FOUND',
 	ENTITY_GROUPS_NOT_FOUND: 'ENTITY_GROUPS_NOT_FOUND',
 	ENTITY_INFORMATION_CREATED: 'ENTITY_INFORMATION_CREATED',
+	TOKEN_INVALID_CODE:'TOKEN_INVALID_CODE',
```

Comment:
@MallanagoudaB "TOKEN_INVALID_MESSAGE": 'Access denied'
TOKEN_INVALID_CODE": 'ERR_TOKEN_INVALID'

update with these values and test